package.json

```diff
@@ -2,12 +2,12 @@
   "name": "shopify-codemod",
   "version": "15.1.0",
   "description": "Custom JSCodeShift transforms to help convert our rusty JavaScript.",
-  "homepage": "https://github.com/Shopify/javascript/tree/master/packages/shopify-codemod",
-  "repository": "https://github.com/Shopify/javascript/tree/master/packages/shopify-codemod",
+  "homepage": "https://github.com/Shopify/shopify-codemod",
+  "repository": "https://github.com/Shopify/shopify-codemod",
   "scripts": {
-    "test": "NODE_PATH=$NODE_PATH:./transforms:./test ../../node_modules/.bin/mocha 'test/**/*.test.js' --compilers js:babel-core/register",
-    "test:watch": "npm run test -- --watch --reporter min",
-    "test:cover": "NODE_PATH=$NODE_PATH:./transforms:./test ../../node_modules/.bin/babel-node ../../node_modules/.bin/isparta cover --root transforms/ --reporter text --reporter html ../../node_modules/.bin/_mocha -- --reporter spec test/**/*.test.js"
+    "test": "NODE_PATH=$NODE_PATH:./transforms:./test mocha 'test/**/*.test.js' --compilers js:babel-core/register",
+    "test:watch": "yarn run test -- --watch --reporter min",
+    "test:cover": "NODE_PATH=$NODE_PATH:./transforms:./test babel-node node_modules/.bin/isparta cover --root transforms/ --report text --report html _mocha -- --reporter spec test/**/*.test.js"
```

Comment:
Switched `--reporter` => `--report` (per [isparta's README](https://github.com/douglasduteil/isparta).  This gives a bit more granular stats in the console.

Can't figure out how to not specify the full path to `isparta` :/